pallets/nfts/src/tests.rs

```diff
@@ -337,6 +448,7 @@ fn mint_should_work() {
 			default_collection_config()
 		));
 		assert_ok!(Nfts::mint(RuntimeOrigin::signed(account(1)), 0, 42, account(1), None));
+		assert_eq!(AccountBalance::<Test>::get(0, account(1)), 1);
```

Comment:
This actually does not makes sense to me because we only add one line to check the account balance and now we have to change all the variables. Which makes it harder to compare with the upstream test case.

Reply:
That is fine.